templates/layouts/main.html

```diff
@@ -7,8 +7,13 @@
 
     {% include 'partials/meta-tags.html' %}
 
-    <!-- <link rel="stylesheet" href="/{{ assets['dist/styles/styles.css'] }}" /> -->
-    <link rel="stylesheet" href="http://localhost:8080/css/styles.css" />
+    <link rel="stylesheet" href="/{{ assets['dist/styles/styles.css'] }}" />
```

Comment:
No, this is the vendor / custom styles. Lines 12-16 adds fec-style. We can change the names if you want.